scripts/check_frontend_coverage.py

```diff
@@ -129,6 +129,8 @@
     'exploration-html-formatter.service.ts',
     'exploration-id-validation.service.ts',
     'exploration-init-state-name.service.ts',
+    'exploration-language-code.service.ts',
+    'exploration-objective.service.ts',
```

Comment:
What about the other two services? Are they not fully covered in this PR?

Reply:
![Screenshot from 2020-04-24 01-41-05](https://user-images.githubusercontent.com/45730556/80144883-eaf57980-85cc-11ea-975c-35b0cd8d2117.png)

all are covered.